index.d.ts

```diff
@@ -1,288 +1,620 @@
-declare module 'react-table' {
-  import { ReactNode, useState } from 'react'
-
-  type StringKey<D> = Extract<keyof D, string>
-  type IdType<D> = StringKey<D> | string
+// TypeScript Version: 3.0
 
-  type SortingRule<D> = {
-    id: IdType<D>
-    desc: boolean
+declare module 'react-table' {
+  import { ReactNode, useState, ComponentType, MouseEvent, Props } from 'react'
```

Comment:
Props is unused.

Reply:
Yeah, not sure how it got there. Had already removed, forgot to push